src/modules/flight_mode_manager/tasks/AutoFollowTarget/FlightTaskAutoFollowTarget.hpp

```diff
@@ -104,6 +114,7 @@ class FlightTaskAutoFollowTarget : public FlightTask
 	bool update() override;
 
 protected:
+	// Follow Perspectives set by the parameter NAV_FT_FS
```

Comment:
you need to check that this 0-360 definition still works with the -pi,pi definition that exists everywhere else in your logic now. otherwise we should redefine this to -180 to 180 for follow perspective (i anyway think this is a bit more intuitive)